fujiji-client/src/components/NavBar/NavBar.spec.jsx

```diff
@@ -1,23 +1,10 @@
-import { render, fireEvent } from '@testing-library/react';
+import { render } from '@testing-library/react';
 
 import NavBar from './NavBar';
 
 describe('NavBar', () => {
   it('should render sign-in and sign-up button when not logged in', () => {
     const { getByText } = render(<NavBar />);
     expect(getByText('Sign In')).toBeInTheDocument();
-    expect(getByText('Sign Up')).toBeInTheDocument();
-  });
-
-  it('should render post and avatar button when logged in', () => {
-    const { getByText, getByLabelText } = render(<NavBar isSignedIn />);
-    expect(getByLabelText('post-listing-button')).toBeInTheDocument();
-    expect(getByLabelText('profile-button')).toBeInTheDocument();
-
-    // test expanded link after clicking profile button
-    fireEvent.click(getByLabelText('profile-button'), { bubble: true });
-    expect(getByText('Home')).toBeInTheDocument();
-    expect(getByText('My Listings')).toBeInTheDocument();
-    expect(getByText('Sign Out')).toBeInTheDocument();
```

Comment:
I think we should still include this test (?) This is testing the behaviour when the avatar icon is clicked. And your tweak have the same behaviour.